src/box/applier.cc

```diff
@@ -1653,6 +1653,7 @@ applier_apply_tx(struct applier *applier, struct stailq *rows)
 			       &replicaset.applier.order_latch);
```

Comment:
Please, let's not use links in the commit title. Looks broken and ugly:

![image](https://github.com/user-attachments/assets/bb3ac507-be09-40ee-9dc3-359f9743c6af)

Reply:
Fixed.